services/mqtt/service.go

```diff
@@ -0,0 +1,186 @@
+package mqtt
+
+import (
+	"fmt"
+	"log"
+	"sync/atomic"
+	"time"
+
+	pahomqtt "github.com/eclipse/paho.mqtt.golang"
+	"github.com/influxdata/kapacitor/alert"
+)
+
+// QoSLevel indicates the quality of service for messages delivered to a
+// broker.
+type QoSLevel byte
```

Comment:
Can you implement the TextUnmarshaler interface on this type so its toml representation is not numeric?